examples/Basic.hs

```diff
@@ -22,18 +22,26 @@ mean xs = sum' xs / fromIntegral (length xs)
 
 main :: IO ()
 main = do
-    handle <- forkServer "localhost" 8000
-    counter <- getCounter "iterations" handle
-    label <- getLabel "args" handle
-    event <- getDistribution "runtime" handle
+    handle <- forkServer "0.0.0.0" 8000
+    baseCounter  <- getCounter "iterations" [] handle
```

Comment:
For example, what would it mean to sum up all the values in a metric if you have "global" counts and indexed counts?